content/attacks/2023-11-18-Kronos.md

```diff
@@ -0,0 +1,39 @@
+---
+date: 2023-11-19
+target-entities: Kronos Research 
+entity-types: 
+   - DeFi
+attack-types:
+   - Private Key Leak
+title: Kronos Research halts trading after $25M API key hack
+loss: 26000000
+---
+
+## Summary
+
+On November 19, 2023, Kronos Research, a Taipei-based cryptocurrency trading and investment firm, was targeted by a hacker who stole over $25 million from the firm's treasury using [unauthorized API keys.](https://cryptonews.com/news/kronos-research-enters-negotiations-with-hacker-after-25-million-cryptocurrency-theft-offers-10-bounty.htm) This breach enabled the attacker to access the company's blockchain wallets and conduct unauthorized transactions. The attack's impact extended beyond Kronos Research, affecting Woo X, an exchange closely affiliated with Kronos Research. As Kronos Research was a major liquidity provider for Woo X, the security incident led to a temporary suspension of certain asset pairs on Woo X due to a liquidity shortage.
+
+## Attackers
+
+The Attacker has not been identified or communicated with Kronos Research since the hack.
+
+## Losses
+
+At the time of the attack, Kronos Research lost around $26 million. According to Lookonchain’s X feed, the following losses occurred:
+
+   - $24.57M in USDT
+   - $488.7 in ETH
+   - $125,056 in USDC
+
+## Timeline
+
+   - **November 18, 2023, 05:05 PM UTC:** Initial funds transfer [begins.](https://etherscan.io/address/0x2b0502fdab4e221dcd492c058255d2073d50a3ae) 
+   - **November 18, 2023, 11:11 PM UTC:** ZachXBT posts a graph with six of the ETH [transactions](https://twitter.com/zachxbt/status/1726015231023796233) on X.
+   - **November 19, 2023, 11:27 AM UTC:** Kronos Research releases a [series](https://twitter.com/ResearchKronos/status/1726203102842466650) of posts on X confirming the hack and stating the firm will internally cover all losses.
+   - **November 28, 2023, 06:30 PM UTC:** Kronos Research offers a [whitehat bounty](https://etherscan.io/idm?addresses=0xad5916c0f641841637bab1a1049224c3cfd5acf0,0x7e1a22655e2a46a5dd8aec2905c298f1d06b8597&type=1) of 10% and promises no legal ramifications if funds are returned by 8:00 UTC on November 30, 2023.
+
+## Security Failure Causes
+
+   - **Compromised API Keys:** Indicates a failure in securing and monitoring API key access.
+   - **Inadequate Anomaly Detection and Response:** The swift execution of the theft points to a lack of effective systems to detect and respond to unauthorized access in real-time.
+   - **Asset Liquidation and Laundering Strategy:** The attackers' method of converting stolen assets and dispersing them across various wallets highlights a need for better transaction monitoring and tracing mechanisms.
```

Comment:
> ChatGPT4 used to update security failures.

This doesn't look like a fundamental IT Security concept.

Reply:
@evgenydmitriev  Security failures updated.